### DISCUSSION TITLE
What is difference between 2ch and 16ch?

### DISCUSSION THREAD

**Discussion by MaxLevs** [ExistentialAudio/BlackHole#332] [General]
I tried to figure it out but I still doesn't understand which version should I install for my MacBook 13'' 2015 with Big Sur

**Comment by high-solutions** [comment]
as far as i know there is not really a difference except the 16 channels supports 16 channels, so you can use 16ch hardware audio interfaces and route every input trough blackhole separate.

**Comment by devinroth** [comment]
Exactly. They are the same except the channel count.

Devin
